tests/common/logout_test.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var App = require('../lib/app');
+var assert = require('assert');
+
+describe('Logout', function() {
+  var app;
+  var loginView;
+  var servicesView;
+  this.timeout(30000);
+
+  before(() => {
+    app = new App();
+    return app.init()
+      .then(defaultView => { loginView = defaultView; });
+  });
+
+  after(() => {
+    return app.stop();
+  });
+
+  it('should logout', () => {
+    return loginView.loginSuccess(12345678).then(function(servicesView) {
```

Comment:
It can be nested, but it's not considered as a good practice. Promises were thought as an improvement of callbacks, and nesting was [one of the pain point](http://callbackhell.com/).

The way to prevent nesting is the following:
```js
return loginView.loginSuccess(12345678)
  .then(servicesView => {
    return servicesView.logoutSuccess(); // promise is returned there and chained to the top level.
  }).then(() => {
    // do something after it's logged out
  })
```

Thanks to arrow functions, this can be shortened to:
```js
return loginView.loginSuccess(12345678)
  .then(servicesView => servicesView.logoutSuccess())
  .then(() => {
    // do something after it's logged out
  })
```